packages/now-php-bridge/build/Dockerfile

```
@@ -1,8 +1,15 @@
-FROM library/centos:6.8
+FROM library/centos:7
```

```suggestion
FROM library/centos:6.8
```

Does this work without changing centos version?